travis/build-and-test.sh

```diff
@@ -34,3 +34,18 @@ make -j$(nproc)
 
 # run all unit tests
 ctest -V
+
+# install libappimage
+if [[ `whoami` != root ]]
```

Comment:
This will horribly break local testing of that file, it is supposed to *not* modify the system. A *big* no-no. Also, doesn't really fit in this PR.

Reply:
Reasons explained int the above answers.

Reply:
Fixed, please next time explain your idea of how something could be improved right away.

Reply:
Sure.